Support local context for webapp bind postgres

chagong opened the pull request:
Support local context for webapp bind postgres. 
* sub-command for it is "az cupertino webapp postgres bind ..."
* Because Azure CLI postgres local context is not supported yet, only webapp name can be saved into local context, we still need to input postgres parameters. 
* But in our extension I've added the code for postgres local context;

chagong committed:
Support local context for webapp bind postgres

dooriya commented on azext_connect/_params.py:
```diff
@@ -33,6 +49,18 @@ def load_arguments(self, _):
         c.argument('password', options_list=['--password', '-pwd'], help='Password of the database. Only valid when auth type is secret')
         c.argument('keyvault', options_list=['--keyvault', '-kvt'], help='Keyvault name in the same resource group.')
 
+    with self.argument_context('cupertino webapp postgres') as c:
```
Since the command contains three sub-groups, please sync with Xin and CLI team to make sure if this is a reasonable command to expose to the end-user for private preview

chagong replied:
@shizn I added you in the reviewer list.

arrownj reviewed (approved):
It's good to me for LocalContextAttribute part.

chagong committed:
verified postgres local context and enable interactive input for
username and password
i Please enter the commit message for your changes. Lines starting

chagong committed:
remove extra space

KennethBWSong commented on azext_connect/custom.py:
```diff
@@ -499,6 +500,31 @@ def bind_webapp(
+        target = _get_target_id(scope, postgres=server, database=database)
+        result = _bind(
+            cmd, subscription, resource_group, name, source,
+            target, authtype, permission, None, None, username, password
```
Client_id and Client_secret are for Service Principal AuthType. It seems that service principal should be supported according to _params.py, thus it may be better not to set them as None.

chagong replied:
Have we already supported SP in the backend code?

KennethBWSong replied:
No, currently we have not support SP yet. I mean the design of cli seems take SP into consideration, and it may be better to follow the pattern.

chagong replied:
Make sense, let me add these 2 parameters.

chagong committed:
Link name will to auto-generated for webap/postgres.
